db/view/view.cc

```diff
@@ -1203,9 +1204,10 @@ future<> mutate_MV(
         service::allow_hints allow_hints,
         wait_for_all_updates wait_for_all)
 {
-    auto fs = std::make_unique<std::vector<future<>>>();
-    fs->reserve(view_updates.size());
-    for (frozen_mutation_and_schema& mut : view_updates) {
+    static constexpr size_t max_concurrent_updates = 128;
```

Comment:
Food for thought for later (I hope you're keeping track ;-)). Consider that a user deletes 100 partitions in parallel, thinking that 100 is reasonable concurrency. But here each of these deletions causes 128 parallel view updates, so we end up with 12,800 concurrent updates. Maybe we should have a more "global" (per table) semaphore for this view-update concurrency, so its *total* concurrency is limited?

Reply:
We do - we have a `_config.view_update_concurrency_semaphore` per table. This limit is just to prevent spawning millions of tasks at once, just to be limited with a semaphore later.